code/game/area/space_station_13_areas.dm

```diff
@@ -534,6 +534,9 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 
 /area/chapel
 	icon_state = "chapel"
+	mood_bonus = 5
+	mood_message = "<span class='nicegreen'>Being in the chapel brings me peace.\n</span>"
```

Comment:
```suggestion
	mood_message = "<span class='nicegreen'>Being in the chapel brings me peace.</span>\n"
```

Reply:
The bar and psych office both also put the `\n` before the `</span>` Should they be changed as well?

Reply:
Mmm, good question. All the mood messages that I've seen from regular moodlets, all have the `\n` outside the span. So yeah, change those too.

Reply:
Should I do that here, or in a separate PR?

Reply:
Do it in a seperate PR, this one's ready to go.